### ISSUE TITLE
Add per project after trigger

### ISSUE THREAD

**Issue by greyhoundforty** [Angelmmiguel/pm#1]
Any chance for a way to trigger per project after actions? Right now it looks like the default is after-all but would be nice to set specific actions per project.

**Comment by Angelmmiguel** [comment]
Hello!

Yes, it is a planned feature, I will add it in a few days :smiley:

**Comment by greyhoundforty** [comment]
Awesome. I'm liking pm a lot so far as a simple project manager

Sent from my iPhone

On Sep 26, 2015, at 4:33 PM, ?ngel M <notifications@github.com<mailto:notifications@github.com>> wrote:


Hello!

Yes, it is a planned feature, I will add it in a few days [:smiley:]

-

**Comment by Angelmmiguel** [comment]
Thanks! I hope it help you.

**Comment by Angelmmiguel** [comment]
I added this functionality to new version. I need to create an "update" process from [version 0.1](https://github.com/Angelmmiguel/pm/releases/tag/v0.1) to this version because I changed the format of `~/.pm/projects` to store the properties of them.

The change is to enclose to projects. Before this feature:

```
pm:/Users/myuser/myproject
irb:/Users/myuser/mysecondproject
```

Now, 

```
pm:/Users/myuser/myproject
\pm
irb:/Users/myuser/mysecondproject
\irb
```

I will add an update process to perform this changes automatically.
